lib/lightstep/span_context.rb

```diff
@@ -3,29 +3,43 @@
 module LightStep
   # SpanContext holds the data for a span that gets inherited to child spans
   class SpanContext
-    attr_reader :id, :trace_id, :trace_id16, :sampled, :baggage
+    attr_reader :id, :trace_id, :trace_id_upper64, :sampled, :baggage
```

Comment:
Could we also alias `:trace_id_upper64` to `:trace_id`?

Reply:
`trace_id` is the lower 64 bits. Do you want an alias for that?